orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -2562,3 +2562,24 @@ def forward(self, input1, input2, input3, input4, input5, input6, input7, input8
     assert len(out_pt) == len(out_ort)
     for x, y in zip(out_pt, out_ort):
         _test_helpers.assert_values_are_close(x, y)
+
+def test_stateless_model():
+    class StatelessModel(torch.nn.Module):
+        def __init__(self):
+            super(StatelessModel, self).__init__()
+
+        def forward(self, x):
+            return x
+
+    device = 'cuda'
+    N, D_in, H, D_out = 32, 784, 500, 10
+    pt_model = StatelessModel().to(device)
+    ort_model = ORTModule(copy.deepcopy(pt_model))
+
+    pt_x = torch.randn(N, D_in, device=device)
```

Comment:
`pt_x` has `device=device`, what happens if this is not specified? It would be good to have this test too

Reply:
I'll add a test for this.